sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```
@@ -59,8 +61,10 @@ def construct_iso8601(start=None, end=None, duration=None):
         elif duration is not None:
```

What's the behavior if user provides start, end and duration?

---

we would take the start and end time - we have documented that all three should not be provided - on retrospect, let me warn here too

---

In fact my question is if three are provided, we would silently ignore duration (current behavior) or give error? :)

---

we now give error :)